### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/65ba9f405b0fa4715d48c612ba769458cee120e8

### CODE REVIEW

File: en/mozorg/about.ftl [hunk 1]
```diff
@@ -28,6 +28,8 @@ about-walking-our-privacy-talk = Walking Our Privacy Talk
 about-when-the-facebook-breach-was = When the { -brand-name-facebook } breach was revealed, { -brand-name-mozilla } had an immediate response – and a { -brand-name-firefox } product to support user privacy.
 about-talking-internet-issues-irl = Talking Internet Issues IRL
 about-in-mozillas-irl-podcast-host = In { -brand-name-mozilla }’s IRL podcast, host Manoush Zomorodi shares real stories of life online and real talk about the future of the Web.
+vision-for-the-web = { -brand-name-mozilla }'s vision for the Web
```
Comment (peiying2) [inline en/mozorg/about.ftl:31]: - Use Curly quote
- Web `Web` should be lower case

File: en/mozorg/about.ftl [hunk 2]
```diff
@@ -28,6 +28,8 @@ about-walking-our-privacy-talk = Walking Our Privacy Talk
...
+read-about-our-vision = Read about our vision for the Web and how we intend to pursue that vision.
```
Comment (peiying2) [inline en/mozorg/about.ftl:32]: `Web` should be lower case

File: en/mozorg/about/shared.ftl [hunk 3]
```diff
@@ -15,3 +15,5 @@ about-shared-our-products = Our Products
 about-shared-software-innovations = Software and other innovations designed to advance our mission.
 about-shared-get-involved = Get Involved
 about-shared-volunteer = Become a volunteer contributor in a number of different areas.
+vision-for-the-web = { -brand-name-mozilla }'s vision for the Web
```
Comment (peiying2) [inline en/mozorg/about/shared.ftl:18]: - Use Curly quote
- Web `Web` should be lower case

File: en/mozorg/about/shared.ftl [hunk 4]
```diff
@@ -15,3 +15,5 @@ about-shared-our-products = Our Products
...
+read-about-our-vision = Read about our vision for the Web and how we intend to pursue that vision.
```
Comment (peiying2) [inline en/mozorg/about/shared.ftl:19]: `Web` should be lower case

**Review by peiying2 — CHANGES_REQUESTED** [review]
- Use curly quote instead of straight.
- `Web` was replaced with lower case `web` at some point. There might be some inconsistency throughout the site, and even this page with. 
- Why we add the same strings in the `about.ftl` and the `about/shared.ftl`? Should we just add the new strings to `about/shared.ftl` only?

**Comment by peiying2** [comment]
@reemhamz

**Comment by reemhamz** [comment]
Thanks for this Peiying! I'm just confirming a few things and will go forward from there.

**Comment by peiying2** [comment]
> Hi Peiying, So I [checked with the team](https://github.com/mozilla/bedrock/issues/11501#issuecomment-1101813945) and they'd like to keep the word `web` as capitalized (`Web`) wherever I placed the strings to translate, primarily because it's seen as a title. So we'll keep those as is. I'll also delete the strings from `about.ftl` since both pages will read `about/shared.ftl`. Thanks for catching that. I'll update the straight quotes to curly quotes also. A pull request will be merged soon for these changes

@reemhamz Thank you for the update. I understand the title case situation. What  about  the `Web` that is in a sentence?

**Comment by pmac** [comment]
@peiying2 @k88hudson asked that we keep `Web` capitalized in https://github.com/mozilla/bedrock/issues/11501

**Comment by flodolo** [comment]
I'm personally not a fan of having a [guideline](https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=FIREFOX&title=Word+List#WordList-web) and ignoring it, but given that's not a guideline explicitly set for mozilla.org, I think we should go ahead.

**Pull request merged**